Listenbrainz/model/Feed/Metadata.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  Metadata.swift
+//  Listenbrainz
+//
+//  Created by Gaurav Bhardwaj on 30/08/23.
+//
+
+import Foundation
+
+
+struct Metadata: Codable {
+    let insertedAt, listenedAt: Int?
+    let listenedAtISO, playingNow: JSONNull?
+    let trackMetadata: FeedTrackMetadata?
+    let userName: User?
+    let created: Int?
+    let relationshipType: String?
+    let userName0: User?
```

Comment:
Something is wrong here. This is not how this should be parsed. Check the data again

Reply:
<img width="1440" alt="Screenshot 2023-09-18 at 3 44 19 PM" src="https://github.com/metabrainz/listenbrainz-ios/assets/106892217/9bc14c83-647a-4dd1-a59f-10c07a6ddc3d">
This I have taken from that tool, I guess there should not be any problem in this.

Reply:
@gauravbhardwaj7 Now I understand why the api call works fine for you. Because it is customed towards `your` feed response only and is not generic to accomodate to my feed api response

Reply:
Yes! , your feed response is different from mine , how should we tackle this?

Reply:
Understand how the data is flowing. The tool is good to get you the boilerplate code but don't rely on it 100% @gauravbhardwaj7 . I hope that makes sense?

Reply:
Yes makes sense!